supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesJobCachingActorHelper.scala

```diff
@@ -19,5 +19,6 @@ trait TesJobCachingActorHelper extends StandardCachingActorHelper {
   lazy val tesConfiguration: TesConfiguration = initializationData.tesConfiguration
 
   lazy val runtimeAttributes = TesRuntimeAttributes(validatedRuntimeAttributes, jobDescriptor.runtimeAttributes, tesConfiguration)
+  override protected def nonStandardMetadata: Map[String, Any] = super.nonStandardMetadata ++ tesJobPaths.azureLogPathsForMetadata
```

Comment:
I love this naming and I'm glad you found the perfect fit to stick some extra stuff